### DISCUSSION TITLE
Cursor pointers

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#500] [Proposals]
Cuándo usar el cursor pointer en los hovers?

siempre que sea un elemento interactivo?
solo para links?

https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b

Yo lo simplificaría y lo usaría en cualquier cosa que el usuario pueda "tocar" / interactuar, aunque eso entra en conflicto con definiciones de cuándo debería usarse el pointer (se supone que solo para links) pero la realidad es que en la mayoría de sitios las cosas que son interactivas terminan usando el pointer

**Comment by aweell** [comment]
En este tema estoy de acuerdo con el autor, el pointer ayuda a describir interacciones que no están disponibles en cualquier elemento sino en un anchor (abrir en una nueva pestaña, ventana, copiar enlace...). 

En la definición de CSS3 UI (https://www.w3.org/TR/css-ui-3/#cursor). Este estilo se asocia directamente a links.

Aquí abro un melón que tiene que ver con el Button link en desktop, visualmente se representa igual ya utilice un `onPress` o un `href`, añadiendo el cursor:"pointer" el usuario no tiene ninguna pista sobre que opciones tiene disponibles cuando haga click secundario, si no es a través del cursor, cual debería se la manera más apropiada de definir visualmente esta distinción? Consideramos necesario que exista esta distinción?

La pista debería ser que la url es mostrada por el navegador en la zona inferior del mismo? Esto lo hacen todos los navegadores? 

Quizás no debería existir un buttonLink con onPress? (Tengo mis dudas, de que esto sea necesario por detalles de implementación)
